admin/database/postgres/migrations/0049.sql

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE project_variables (
+    id UUID NOT NULL DEFAULT uuid_generate_v4() PRIMARY KEY,
+    project_id UUID NOT NULL REFERENCES projects(id) ON DELETE CASCADE,
+    name TEXT NOT NULL,
+    value BYTEA NOT NULL,
+    -- Encryption key ID if the value is encrypted
+    value_encryption_key_id TEXT NOT NULL DEFAULT '',
+    -- Environment it belongs to ("production" or "development").
```

Comment:
The built-in environments are currently called `"prod" or "dev"` (not fully spelled out)